src/hypervisor.c

```diff
@@ -344,6 +361,8 @@ cc_oci_expand_cmdline (struct cc_oci_config *config,
 		{ "@NETDEV_PARAMS@"     , netdev_params              },
 		{ "@NETDEVICE@"         , net_device_option          },
 		{ "@NETDEVICE_PARAMS@"  , net_device_params          },
+		{ "@AGENT_CTL_SOCKET@"  , agent_ctl_socket           },
+		{ "@AGENT_TTY_SOCKET@"  , agent_tty_socket           },
```

Comment:
Could you add these to the "Variable Expansion" section in `README.rst` please.

Aside: It looks like that file is also missing the following (not required for this PR but we should add these):

- `@KERNEL_NET_PARAMS@`
- `@NETDEV@`
- `@NETDEV_PARAMS@`
- `@NETDEVICE@`
- `@NETDEVICE_PARAMS@`